localizations/zh_CN.json

```diff
@@ -220,8 +220,8 @@
     "latent nothing": "潜空间数值零",
     "Inpaint at full resolution": "全分辨率局部重绘",
     "Inpaint at full resolution padding, pixels": "预留像素",
-    "Process images in a directory on the same machine where the server is running.": "使用服务器主机上的一个目录，作为输入目录处理图像",
-    "Use an empty output directory to save pictures normally instead of writing to the output directory.": "使用一个空的文件夹作为输出目录，而不是使用默认的 output 文件夹作为输出目录",
+    "Process images in a directory on the same machine where the server is running.": "处理服务器主机上某一目录里的图像",
+    "Use an empty output directory to save pictures normally instead of writing to the output directory.": "使用一个空目录作为输出，而不是使用默认的output目录",
```

Comment:
```suggestion
    "Use an empty output directory to save pictures normally instead of writing to the output directory.": "使用另一个空目录作为输出，而非使用设定里默认的输出目录",
```

Reply:
这个 output 是文件夹的名字吧？我以为是这样，所以保留了字母的形式

Reply:
哦，那个目录叫 outputs ……
这样的话感觉它原文也有点怪（

Reply:
我一开始也以为是文件夹的名字，然后一想他那个文件夹反正是可以在自定义里面改的。估计只是单纯说输出文件夹吧（）